packages/@aws-cdk/aws-apigateway/test/authorizers/integ.request-authorizer-identity-source.ts

```diff
@@ -0,0 +1,41 @@
+import * as lambda from '@aws-cdk/aws-lambda';
+import { App, Stack } from '@aws-cdk/core';
+import * as path from 'path';
+import { MockIntegration, PassthroughBehavior, RestApi } from '../../lib';
+import {RequestAuthorizer} from '../../lib/authorizers';
+
+// Against the RestApi endpoint from the stack output, run
+// `curl -s -o /dev/null -w "%{http_code}" <url>` should return 401
+// `curl -s -o /dev/null -w "%{http_code}" -H 'Authorization: deny' <url>` should return 403
```

Comment:
Both of these return 200, and not 401 and 403.

Can we configure the authorizer lambda so it recognizes the header (or any other request variable supported by the request authorizer) to throw a 4xx?